ui/app/components/project-config-variables/list-item.hbs

```diff
@@ -0,0 +1,149 @@
+              <XToggle
+
+                @showLabels={{true}}
+                @name="name-is-path"
+                @value={{this.variable.nameIsPath}}
+                @size="small"
+                @onToggle={{fn (mut this.variable.nameIsPath)}} as |toggle|>
+                <toggle.switch
+                  @showLabels={{false}}
+                  @name="name-path"
+                  />
+                <toggle.label
+                  data-test-config-variables-name-is-path-toggle
+                  for="name-is-path"
+                  @value={{not this.variable.nameIsPath}}>
+                  <span class="pds-toggleLabel">
+                    {{t "form.config_variables_settings.nameIsPath.label"}}
+                  </span>
+                </toggle.label>
+              </XToggle>
+            </div>
+          </div>
+          <div class="pds-formField">
+            <div class="pds-formFieldToggle">
+              <XToggle
+
+                @showLabels={{true}}
+                @name="internal"
+                @value={{this.variable.internal}}
+                @size="small"
+                @onToggle={{fn (mut this.variable.internal)}} as |toggle|>
+                <toggle.switch
+                  @showLabels={{false}}
+                  @name="internal"
+                  />
+                <toggle.label
+                  data-test-config-variables-internal-toggle
+                  for="internal"
+                  @value={{not this.variable.internal}}>
+                  <span class="pds-toggleLabel">
+                    {{t "form.config_variables_settings.internal.label"}}
+                  </span>
+                </toggle.label>
+              </XToggle>
+            </div>
+          </div>
+        </fieldset>
+        <hr/>
+        <div class="card-footer">
+          <Pds::ButtonSet>
+            <Pds::Button
+              data-test-config-variables-edit-save
+              @variant="primary"
+              @type="submit">
+              {{t "form.config_variables_settings.button_submit"}}
+            </Pds::Button>
+            <Pds::Button
+              data-test-config-variables-edit-cancel
+              @variant="secondary"
+              {{on "click"
+                (if this.isCreating this.cancelCreate this.cancelEdit)
+              }}>
+              {{t "form.config_variables_settings.button_cancel"}}
+            </Pds::Button>
+          </Pds::ButtonSet>
+        </div>
+      </div>
+    </form>
+  {{else}}
+    <div class="variables--list-item">
+      <span class="variables--list-item-name" data-test-config-variables-var-name>{{this.variable.name}}</span>
+      <span class="variables--list-item-value" data-test-config-variables-var-value>
+        {{#if this.variable.pb_static}}
+          <pre><code>{{this.variable.pb_static}}</code></pre>
+        {{/if}}
+        {{#if this.variable.dynamic.from}}
+          <pre><code>dynamic via {{this.variable.dynamic.from}}</code></pre>
```

Comment:
This is maybe for a follow-up PR, but I feel like we could display more information here:
![CleanShot 2021-09-03 at 17 38 08@2x](https://user-images.githubusercontent.com/34030/132032070-27ed7fdc-9c7c-4777-8909-aa0aadd0d53f.png)

Reply:
Yes, we could display more, but for clarification, this text replicates exactly what is displayed in the CL

Reply:
Good to know